gtfs/spec/en/reference.md

```diff
@@ -209,244 +211,183 @@ The example below is valid, with distinct blocks every day of the week.
-|   |  | | A value of **1** indicates that service is available for all Thursdays in the date range. (The date range is specified using the **start_date** and **end_date** fields.) |
-|   |  | | A value of **0** indicates that service is not available on Thursdays in the date range. |
-|   |  | | **Note:** You may list exceptions for particular dates, such as holidays, in the [calendar_dates.txt](#calendar_datestxt) file. |
-|  friday | Enum | **Required** | The friday field contains a binary value that indicates whether the service is valid for all Fridays. |
-|   |  | | A value of **1** indicates that service is available for all Fridays in the date range. (The date range is specified using the **start_date** and **end_date** fields.) |
-|   |  | | A value of **0** indicates that service is not available on Fridays in the date range. |
-|   |  | | **Note:** You may list exceptions for particular dates, such as holidays, in the [calendar_dates.txt](#calendar_datestxt) file |
-|  saturday | Enum | **Required** | The saturday field contains a binary value that indicates whether the service is valid for all Saturdays. |
-|   |  | | A value of **1** indicates that service is available for all Saturdays in the date range. (The date range is specified using the **start_date** and **end_date** fields.) |
-|   |  | | A value of **0** indicates that service is not available on Saturdays in the date range. |
-|   |  | | **Note:** You may list exceptions for particular dates, such as holidays, in the [calendar_dates.txt](#calendar_datestxt) file. |
-|  sunday | Enum | **Required** | The sunday field contains a binary value that indicates whether the service is valid for all Sundays. |
-|   |  | | A value of **1** indicates that service is available for all Sundays in the date range. (The date range is specified using the **start_date** and **end_date** fields.) |
-|   |  | | A value of **0** indicates that service is not available on Sundays in the date range. |
-|   |  | | **Note:** You may list exceptions for particular dates, such as holidays, in the [calendar_dates.txt](#calendar_datestxt) file. |
-|  start_date | Date | **Required** | The **start_date** field contains the start date for the service.  The start_date field's value should be in YYYYMMDD format. |
-|  end_date | Date | **Required** | The **end_date** field contains the end date for the service. This date is included in the service interval.  The **end_date** field's value should be in YYYYMMDD format. |
+|  service_id | ID | **Required** | Identifies a set of dates when service is available for one or more routes. |
+|  monday | Enum | **Required** | Indicates whether the service has been added or removed for all Mondays in the date range specified by the `start_date` and `end_date` fields. Note that exceptions for particular may be listed in [calendar_dates.txt](#calendar_datestxt). Valid values are: |
```

Comment:
Change: "Indicates whether the service operates on all Mondays in the date range specified by the `start_date` and `end_date` fields."

[Justification: I'm concerned about the use of the words "added" and "removed" here, because they have a connotation of an exception to the base schedule, which is what calendar_dates.txt is for.]